cli/cmd/integration.go

```diff
@@ -552,6 +555,26 @@ func reflectIntegrationData(raw api.RawIntegration) [][]string {
 
 		return out
 
+	case api.QRadarChannelIntegration.String():
+
+		var iData api.QRadarChannelData
+		err := mapstructure.Decode(raw.Data, &iData)
+		if err != nil {
+			cli.Log.Debugw("unable to decode integration data",
+				"integration_type", raw.Type,
+				"raw_data", raw.Data,
+				"error", err,
+			)
+			break
+		}
+		out := [][]string{
+			[]string{"QRADAR HOST PORT", fmt.Sprint(iData.HostPort)},
```

Comment:
I'm not sure about adding `QRADAR` in the port and URL... 🤔 but I know why you did it so, up to you to leave it or not.